lib/cylc/gui/updater.py

```diff
@@ -427,23 +416,14 @@ def update(self):
         else:
             # Got suite data.
             self.version_mismatch_warned = False
-            if (self.status == SUITE_STATUS_STOPPING and
-                    self.info_bar.prog_bar_can_start()):
-                gobject.idle_add(
-                    self.info_bar.prog_bar_start, self.status)
-            if (self.is_reloading and
-                    self.info_bar.prog_bar_can_start()):
-                gobject.idle_add(
-                    self.info_bar.prog_bar_start, "reloading")
-            if (self.info_bar.prog_bar_active() and
-                    not self.is_reloading and
-                    self.status not in [SUITE_STATUS_STOPPING,
-                                        SUITE_STATUS_INITIALISING]):
-                gobject.idle_add(self.info_bar.prog_bar_stop)
-            if summaries_changed or err_log_changed:
-                return True
+            if self.status in [
+                    SUITE_STATUS_INITIALISING, SUITE_STATUS_STOPPING]:
+                gobject.idle_add(self.info_bar.prog_bar_start, self.status)
+            elif self.is_reloading:
+                gobject.idle_add(self.info_bar.prog_bar_start, "reloading")
             else:
-                return False
+                gobject.idle_add(self.info_bar.prog_bar_stop)
```

Comment:
No need to call `self.info_bar.prog_bar_can_start()` and `self.info_bar.prog_bar_active()` as `self.info_bar.prog_bar_start` and `self.info_bar.prog_bar_stop` would do their own checks.